package.json

```diff
@@ -10,7 +10,7 @@
   },
   "homepage": "https://github.com/PyMedusa/Medusa#readme",
   "scripts": {
-    "test": "yarn lint",
+    "test": "yarn lint && yarn test-api",
```

Comment:
Check slack.

Reply:
I think it was done in a hurry to show rato the issue.